compiler/util/parseJavaScript.js

```diff
@@ -219,19 +218,15 @@ function parseExpression(src, builder, isExpression) {
         jsAST = esprima.parse(src);
     } catch(e) {
         var errorIndex = e.index;
-
-        var errorMessage = e.toString();
+        var errorMessage = '\n' + e.description;
         if (errorIndex != null && errorIndex >= 0) {
-            var srcCharProps = charProps(src);
             if (isExpression) {
                 errorIndex--; // Account for extra paren added to start
             }
-
-            let line = srcCharProps.lineAt(errorIndex)+1;
-            let column = srcCharProps.columnAt(errorIndex)+1;
-            errorMessage += ' [Line ' + line + ', Col: ' + column + ']';
```

Comment:
Hi @crsandeep, any particular reason to remove line/col numbers here? They're probably still useful, even with the additional error message details.